locales/es.json

```diff
@@ -29,35 +29,35 @@
+  "footer-col-2-1": "<a href='https://www.freecodecamp.org/espanol/news/diseno-ux-convertirse-en-un-disenador-de-experiencia-de-usuario/'>UX</a>",
+  "footer-col-2-2": "<a href='https://www.freecodecamp.org/espanol/news/pensamiento-de-diseno-emprendedores-desarrolladores/'>Proceso de Diseño</a>",
+  "footer-col-2-3": "<a href='https://www.freecodecamp.org/espanol/news/lista-de-numeros-primos-una-grafica-con-todos-los/'>Números Primos</a>",
+  "footer-col-2-4": "<a href='https://www.freecodecamp.org/espanol/news/product-design-explicado-en-un-espanol-sencillo/'>Diseño de Producto</a>",
   "footer-col-2-5": "<a href='https://www.freecodecamp.org/news/what-is-digital-design-and-why-does-it-matter/'>Digital Design</a>",
-  "footer-col-2-6": "<a href='https://www.freecodecamp.org/news/best-coding-games-online-adults-learn-to-code/'>Coding Games</a>",
+  "footer-col-2-6": "<a href='https://www.freecodecamp.org/espanol/news/los-mejores-juegos-online-para-aprender-programar/'>Juegos de Código</a>",
   "footer-col-2-7": "<a href='https://www.freecodecamp.org/news/svm-machine-learning-tutorial-what-is-the-support-vector-machine-algorithm-explained-with-code-examples/'>SVM</a>",
   "footer-col-2-8": "<a href='https://www.freecodecamp.org/news/javascript-foreach-how-to-loop-through-an-array-in-js/'>JavaScript forEach</a>",
   "footer-col-2-9": "<a href='https://www.freecodecamp.org/news/google-bert-nlp-machine-learning-tutorial/'>Google BERT</a>",
-  "footer-col-2-10": "<a href='https://www.freecodecamp.org/news/sql-create-table-statement-with-example-syntax/'>Create Table SQL</a>",
-  "footer-col-3-1": "<a href='https://www.freecodecamp.org/news/responsive-web-design-how-to-make-a-website-look-good-on-phones-and-tablets/'>Responsive Web Design</a>",
-  "footer-col-3-2": "<a href='https://www.freecodecamp.org/news/svg-basics-what-are-scalable-vector-graphics-and-how-do-you-use-them/'>What Is an SVG File?</a>",
-  "footer-col-3-3": "<a href='https://www.freecodecamp.org/news/pdf-password-remover-guide-how-to-remove-password-protection-from-a-pdf/'>PDF Password Remover</a>",
-  "footer-col-3-4": "<a href='https://www.freecodecamp.org/news/what-is-a-pdf-file-and-how-do-you-open-it-solved/'>What Is a PDF?</a>",
+  "footer-col-2-10": "<a href='https://www.freecodecamp.org/espanol/news/create-table-sql-server-con-ejemplo/'>Create Table SQL</a>",
+  "footer-col-3-1": "<a href='https://www.freecodecamp.org/espanol/news/diseno-web-responsive-como-hacer-que-un-sitio-web-se-vea-bien-en-telefonos-y-tabletas/'>Diseño Web Responsive</a>",
+  "footer-col-3-2": "<a href='https://www.freecodecamp.org/espanol/news/que-es-svg-explicacion-de-etiquetas-imagenes/'>¿Qué es un archivo SVG?</a>",
+  "footer-col-3-3": "<a href='https://www.freecodecamp.org/espanol/news/que-es-un-archivo-pdf-como-lo-abres/'>PDF Password Remover</a>",
+  "footer-col-3-4": "<a href='https://www.freecodecamp.org/news/what-is-a-pdf-file-and-how-do-you-open-it-solved/'>¿Qué es un Archivo PDF?</a>",
```

Comment:
Should this link point to an `espanol` article?

Reply:
Yes 🤦🏽